aiida/cmdline/params/options/__init__.py

```diff
@@ -393,3 +393,23 @@ def active_process_states():
 
 CONFIG_FILE = ConfigFileOption(
     '--config', help='Load option values from configuration file in yaml format.')
+
+IDENTIFIER = OverridableOption(
+    '-id', '--identifier', 'identifier',
```

Comment:
We should stick to the rule of using only a single letter for short form options i.e. `-id` -> `-i` or something like that